payroll/views.py

```diff
@@ -240,9 +231,13 @@ def composition_data(self):
         composition_json = []
         percentage_tracker = 0
 
+        unit_base_pay, unit_extra_pay = self.get_entity_payroll()
+        budget = unit_base_pay + unit_extra_pay
+
         for i, value in enumerate(top_departments):
+            value['percentage'] = (float(value['total_expenditure']) / budget) * 100
```

Comment:
IMO, it'd be a little cleaner to store this in a variable, rather than updating `value`, like:

```python
proportion = ...
...
composition_json.append({
    ...
    'data': [proportion]
    ...
})
```